pulsar-broker/src/test/java/org/apache/pulsar/broker/auth/MockedPulsarServiceBaseTest.java

```diff
@@ -708,5 +708,13 @@ public static class ServiceProducer {
         private PersistentTopic persistentTopic;
     }
 
+    protected void sleepSeconds(int seconds){
+        try {
+            Thread.currentThread().sleep(1000 * seconds);
+        } catch (InterruptedException e) {
+            throw new RuntimeException(e);
```

Comment:
```suggestion
            Thread.currentThread().interrupt();
```
I think it's better to do `Thread.currentThread().interrupt()` instead.

Reply:
Sure, fixed it and added a WARN log to help troubleshoot